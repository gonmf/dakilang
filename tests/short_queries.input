
month('January').
month('February').
month('March').
month(name)!
month(name)?
