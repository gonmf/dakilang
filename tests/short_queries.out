>
> month('January').
> month('February').
> month('March').
> month(name)!
name = 'January'

> month(name)?
name = 'January'

name = 'February'

name = 'March'

